homeassistant/components/unmanic/entity.py

```diff
@@ -0,0 +1,32 @@
+"""Unmanic entity class."""
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, NAME
+
+
+class UnmanicEntity(CoordinatorEntity):
+    """Class to represent an Unmanic entity."""
+
+    def __init__(self, coordinator, config_entry, device_id):
+        """Initialize."""
+        self.config_entry = config_entry
+        self._device_id = device_id
+        super().__init__(coordinator)
```

Comment:
```suggestion
        super().__init__(coordinator)
        self.config_entry = config_entry
        self._device_id = device_id
```

You don't need to have super() at the bottom, so I think it is preferred to have it on the top.